### DISCUSSION TITLE
Tag Objects Job Is Disabled, Why?

### DISCUSSION THREAD

**Discussion by andrejohansson** [immich-app/immich#6223] [Q&A] [answered]
![image](https://github.com/immich-app/immich/assets/2828428/ce03bbc3-95fd-428e-a865-a92c3455b200)

Why is this job disabled and what can I do about it?

- Alternatively, where can I find information about why its disabled (or any other job failure).

I'm running on Immich 1.91.4

I found this issue https://github.com/immich-app/immich/issues/5765 but it relates to Raspberry PI, my server is a normal intel i7 cpu server.

**Comment by bo0tzz** [comment, marked as answer]
The object tagging model we used wasn't very good, and the tags weren't really used anywhere. The whole job and model will be removed in the next release.

**Reply by andrejohansson** [reply]
understood, thank you! 

So, I guess this is not the functionality that enables me to search in natural language "images containing a blue car"
